Makefile

```
@@ -284,6 +284,7 @@ integration-test:
 .PHONY: integration-test-docker
 integration-test-docker:
 	scripts/dev_scripts/integration_tests_docker.sh $(REPO_PATH) scripts/release_scripts/run_macaron.sh
+	scripts/dev_scripts/test_run_macaron_sh.py
```

```suggestion
        scripts/dev_scripts/test_run_macaron_sh.py
	scripts/dev_scripts/integration_tests_docker.sh $(REPO_PATH) scripts/release_scripts/run_macaron.sh
```
The `test_run_macaron_sh.py` is shorter, I think we should place it first to exit early when there are errors?

---

Instead of adding the run command here, can you please call this as a test from `integration_tests_docker.sh`?

---

I have moved `test_run_macaron_sh.py` to the `integration_tests_docker.sh` script, as suggested by Behnaz. See 87bfcfec300f31e73f0a62b479c1206b4ddaecf4.